python/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -269,6 +272,13 @@ def constructExtraFiles(self, job):
 
         folder_dir = master_job.getInputWorkspace(create=True).getPath()
 
+        extraopts = ' '
```

Comment:
I do see your point about surreptitiously adding extra lines to the options - perhaps we should add a metadata flag to the job?